src/c++/vdo/tests/RecoveryJournal_t2.c

```diff
@@ -453,6 +485,12 @@ static void attemptRebuild(CorruptionType  corruption,
   }
 }
 
+/**********************************************************************/
+static void testBasicRecovery(void)
+{
```

Comment:
Similar to the above, maybe "testRebuildFullBlocks"?

Reply:
Fixed